.travis.yml

```diff
@@ -17,5 +17,4 @@ before_script:
 script:
   - docker run --cap-add SYS_ADMIN -v $TRAVIS_BUILD_DIR:/opt/openqa -v /var/run/dbus:/var/run/dbus --env-file <(env | grep TRAVIS) -e TRAVIS=true -e FULLSTACK -e UITESTS -e SCHEDULER_FULLSTACK -e GH_PUBLISH dasantiago/openqa-tests make travis-codecov
 after_failure:
-  - cat /tmp/openqa-debug.log
-after_script: yes
+  - cat openqa-debug.log
```

Comment:
The openqa-debug.log will be inside the container. Travis won't be able to access it.

Reply:
not even in the mounted path?

Reply:
(which might not be . but /opt/openqa)

Reply:
in that case yes :-)

Reply:
So use /opt/openqa within Makefile?

Reply:
Now that's really the current directory.

Reply:
[The documentation only says:](https://docs.travis-ci.com/user/environment-variables/)

> TRAVIS_BUILD_DIR: The absolute path to the directory where the repository being built has been copied on the worker.

[However a quick search on the internet seems to show that you're right.](https://stackoverflow.com/a/17683229)